### DISCUSSION TITLE
adding date published and last visited to metadata?

### DISCUSSION THREAD

**Discussion by AngieFSutton** [nianeyna/ao3downloader#184] [Q&A]
Is there a way to add in the metadata for date posted and last visited when using the "get all work links from an ao3 listing (saves links only)" option? I've tried searching the best I can, and it doesn't SEEM to be answered yet.

**Comment by nianeyna** [comment]
I actually can't do date posted because the create date is in fact not shown anywhere in the list view on ao3. which is weird, but oh well. I could add date last updated though! Last visited should be doable as well. I'll go ahead and convert this discussion to an issue, which you can track to find out when I've made the changes.

**Reply by AngieFSutton** [reply]
Date posted isn't available? Huh. But thanks! You rock like the proverbial hurricane, and I hope your 2025 is awesome!
